install-dev/theme/views/license.php

```diff
@@ -32,11 +32,20 @@
   <?php echo $this->getTemplate('license_content'); ?>
 </div>
 
-<div>
+<div class="clearfix">
   <input type="checkbox" id="set_license" class="required" name="licence_agrement" value="1" style="vertical-align: middle;float:left" <?php if ($this->session->licence_agrement) { ?>checked="checked"<?php } ?> />
   <div style="float:left;width:600px;margin-left:8px">
     <label for="set_license">
       <strong><?php echo $this->translator->trans('I agree to the above terms and conditions.', [], 'Install'); ?></strong>
     </label>
   </div>
 </div>
+<div class="infosBlock" style="margin-top: 10px">
+    <h4><?php echo $this->translator->trans('Legal information', [], 'Install'); ?></h4>
+    <p>
+        <?php echo $this->translator->trans(
+        'Know that the store url will be collected. If you want more informations, see this page:', [], 'Install'
```

Comment:
```suggestion
        'Know that some of the store's public and technical data may be collected. If you want more information, see this page:', [], 'Install'
```

What if it'd be URL and PHP in the future? I think keeping the details on the project's website is better.